_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,238 @@
+---
+name: iframe has a unique accessible name
+
+description: |
+ The accessible name for each iframe is unique
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any two ore more `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree) and have the same [non-empty](#non-empty) [accessible name](#accessible-name), within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees), where the `src` attributes refer to different pages.
+
+### Expectation
+
+For each set of target elements, the `iframes` fulfill the same purpose indicated by the [accessible names](#accessible-name).
+
+**Note:** 
+- An `iframe` creates an embedded browsing context. The content of this browsing context is not part of the [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) that it is embedded in.
+
+## Assumptions
+
+There are no good use cases for using the same [accessible name](#accessible-name) for iframes with different sources.
```

Comment:
This assumption is no longer needed and would cause confusion now that this rule actually allows for using the same name (if the expectation that the iframes fulfill the same purpose is met).

My suggestion would be to remove the assumption.